GitUI/CommandsDialogs/FormCreateBranch.cs

```diff
@@ -34,9 +35,9 @@ public FormCreateBranch(GitUICommands commands, ObjectId? objectId, string? newB
         {
             InitializeComponent();
 
-            // work-around the designer bug that can't add controls to FlowLayoutPanel
-            ControlsPanel.Controls.Add(Ok);
-            AcceptButton = Ok;
+            var w = Width;
+            var h = MainPanel.PreferredSize.Height + ControlsPanel.PreferredSize.Height + (Size.Height - ClientSize.Height);
+            MinimumSize = new Size(w, h);
```

Comment:
This is a tactical fix. I'll think how it could be generalized, and send a proposal in a follow up PR.